css/03-generic.css

```diff
@@ -144,7 +148,7 @@ input[type="submit"]:hover {
 .site-header.featured-image {
 	margin-bottom: 0rem;
 }
-/* de-hyphenate large pullquotes */
-.wp-block-pullquote .alignwide, .wp-block-pullquote .alignwide blockquote, .wp-block-pullquote .alignfull, .wp-block-pullquote .alignfull blockquote {
-	hyphens: manual;
-}
+
+.entry-content .wp-block-image img {
+	margin-bottom: 2rem;
```

Comment:
@slambert Was this tested on images with captions? Seems like it might add too much space between the caption and the image.

Reply:
It works on images with captions, but I'm going to remove this also. When I take it out, I don't see what I was fixing anymore. It may have been with the hr or separator, which could be fixed in other ways (adding margins to the hr). I'll just keep an eye out if the problem pops up again.